app/assets/stylesheets/style.scss

```diff
@@ -5,12 +5,13 @@
 }
 
```

Comment:
Just a general comment made previously around if we need these changes or if we can use standard elements already available in the templates.